problems/merge_sort.rb

```diff
@@ -1,5 +1,5 @@
+# Write an `Array#merge_sort` method; it should not modify the original array.
```

Comment:
Let's blacklist `sort` and `sort_by` for this method (dangerous and non-dangerous). Additionally for all methods we blacklist we need to explicitly state as such in the problem description. So add a `before(:each)` hook to test that these methods aren't called on *any instance* of an Array (`expect_any_instance_of(Array)`) and a note to this problem description as follows:

"\*\*Do NOT use the built-in `Array#sort` or `Array#sort_by` methods in your implementation\*\*"